sys/include/net/gnrc/ipv6/nc.h

```diff
@@ -143,6 +146,15 @@ typedef struct {
      */
     vtimer_t nbr_adv_timer;
 
+#ifdef MODULE_GNRC_SIXLOWPAN_ND
+    vtimer_t rtr_sol_timer; /**< Retransmission timer for unicast router solicitations */
+#endif
+#ifdef MODULE_GNRC_SIXLOWPAN_ND_ROUTER
+    vtimer_t type_timeout;                  /**< Timer for type transissions */
+    eui64_t eui64;                          /**< the unique EUI-64 of the neighbor (might be
```

Comment:
Ask the RFC writer... the [RFC 6775](https://tools.ietf.org/html/rfc6775#section-6.5.3) makes it very clear, that the EUI-64 should be stored in addition to the link-local address, since it is used to uniquely identify the interface for DAD (which [is NOT performed](https://tools.ietf.org/html/rfc6775#section-3.1) if the address is based on the EUI-64 btw) and other stuff.